websites/B/Blob Project/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Xyszx",
+      "id": "593774699654283265"
+    },
+    "url": "blob-project.com",
+    "description": {
+      "en": "Hello, welcome to Blob Project. We are from Jakarta, Indonesia. We create an amazing project and are trusted by thousands of people."
+    },
+    "service": "Blob Project",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/xUh0qSN.png",
+    "thumbnail": "https://i.imgur.com/2m91rBI.png",
+    "color": "#7289DA",
```

Comment:
thumbnail should be a promotional banner or a screenshot of the website homepage.

Reply:
Done.